libs/barista-components/stacked-series-chart/README.md

```diff
@@ -0,0 +1,171 @@
+| `nodes` | `DtStackedSeriesChartNode[]` | Array of node for the current series. |
+
+### DtStackedSeriesChartNode
+
+This `DtStackedSeriesChartNode` holds the information for every node in a given
+series.
+
+| Name    | Type                | Optional | Description                                                                                             |
+| ------- | ------------------- | -------- | ------------------------------------------------------------------------------------------------------- |
+| `label` | `string`            | No       | Name of the node to be shown.                                                                           |
+| `value` | `number`            | No       | Numeric value used to calculate the slices.                                                             |
+| `color` | `DtColors | string` | Yes      | Color to be used. Fallback to [sorted chart colors](/resources/colors/chartcolors#sorted-chart-colors). |
+
+### DtStackedSeriesChartTooltipData
+
+The context of the overlay will be set to DtStackedSeriesChartTooltipData object
+containing useful information that can be used inside the overlay's template
+
+| Name            | Type                       | Description                                                      |
+| --------------- | -------------------------- | ---------------------------------------------------------------- |
+| `origin`        | `DtStackedSeriesChartNode` | Node passed by user in `series` array.                           |
+| `valueRelative` | `number`                   | Numeric percentage value based on this node vs sum of top level. |
+| `color`         | `DtColors | string`        | Color for this node in this state.                               |
+| `visible`       | `boolean`                  | If node is visible in the stacked-series-chart.                  |
+| `selected`      | `boolean`                  | If node is currently selected.                                   |
+| `width`         | `string`                   | Current width in percentage given only the visible nodes.        |
+
+### DtStackedSeriesChartLegend
+
+This `DtStackedSeriesChartLegend` holds the information for every legend item so
+color and visibility is unified among same chart but also distributed charts
+(i.e. multiple charts in a table).
+
+| Name      | Type                | Description                                |
+| --------- | ------------------- | ------------------------------------------ |
+| `label`   | `string`            | Label of the node.                         |
+| `color`   | `DtColors | string` | Color to be used based on nodes and theme. |
+| `visible` | `boolean`           | Whether it should be visible.              |
+
+## Examples
+
+### Fill mode
+
+<ba-live-example name="DtExampleStackedSeriesChartFilled" fullwidth></ba-live-example>
+
+### Single selectable stacked bar chart
+
+For 3 nodes or less theme is applied
```

Comment:
This comment does not really add any additional value in this section. Can we remove it?